models/adlinks/position.go

```diff
@@ -0,0 +1,7 @@
+package adlinks
+
+const (
```

Comment:
These would be used for `AdLink` position

Reply:
How about moving this and type to `adlink.go` and put the comment for them? I think they are part of adlink so we can get rid of an unnecessary directory. If agree please check other files too.

Reply:
@gutakk moved to model in [416f1df](https://github.com/carryall/go-google-scraper-challenge/pull/42/commits/416f1dfd8f048d5955ab9691f6d0e0a2e3f2af7c)